src/components/molecules/tables/eventTable/eventTable.tsx

```diff
@@ -65,12 +131,44 @@ const EventTable = () => {
       header: 'Edit',
       type: 'string',
     },
+    {
+      cell: (cellValues) => {
+        const { eid } = cellValues;
+        return (
+          <>
+            <Icon
+              size={2}
+              type="trash"
+              onClick={() => {
+                openDeleteColumn(eid);
+              }}
+            />
+          </>
+        );
+      },
+      header: 'Delete',
+    },
   ];
 
   return (
     <>
       <div className={styles.form}>
         <Table columns={columns} data={events}></Table>
+        {deleteModal && (
+          <Modal
+            title={`Are you sure you want to delete ${selectedEvent?.title}?`}
+            setIsOpen={setOpenDeleteModal}
+            minWidth={45}>
+            <ConformationBox
+              onAccept={() => {
+                setConfirmed(true);
+              }}
+              onDecline={() => {
```

Comment:
This could be made optional and be removed here when the other comments are adressed

Reply:
I still thing onDecline needs to be required as the modal needs to be closed onDecline

Reply:
Yes of course that makes sense🙌

Reply:
Almost worked, however the modal setter can not set the confirmed state back to undefined as it only set boolean values. I think adding a onClick prop to only solve this case is not suitable